app/components/dossiers/annuaire_education_component.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+class Dossiers::AnnuaireEducationComponent < ApplicationComponent
+  attr_reader :champ
+
+  def initialize(champ:)
+    @champ = champ
+  end
+
+  def call
+    render Dossiers::ExternalChampComponent.new(data:, details:, source:)
+  end
+
+  private
+
+  def data
+    [
+      ['Nom de l’établissement', champ.data['nom_etablissement']],
+      ['L’identifiant de l’etablissement', champ.data['identifiant_de_l_etablissement']],
+      ['SIREN/SIRET', champ.data['siren_siret']]
+    ]
+  end
+
+  def details
+    [
+      ['Commune', commune],
+      ['Académie', "#{champ.data['libelle_academie']} (#{champ.data['code_academie']})"],
+      ['Nature de l’établissement', "#{champ.data['libelle_nature']} (#{champ.data['code_nature']})"],
+      ['Type de contrat privé', type_de_contrat],
+      ['Nombre d’élèves', champ.data['nombre_d_eleves']],
+      ['Adresse', adresse],
+      ['Téléphone', champ.data['telephone']],
+      ['Email', champ.data['mail']],
+      ['Site internet', champ.data['web']]
+    ]
+  end
+
+  def commune
+    if champ.data['code_commune'].present?
+      "#{champ.data['nom_commune']} (#{champ.data['code_commune']})"
+    else
+      chang.data['nom_commune']
```

Comment:
```suggestion
      champ.data['nom_commune']
```
(hum ça me remarquer dire qu'on a pas de tests systèmes là dessus, alors qu'il a un gros volume de dossiers)


Aussi petit update : on a des établissements sans `nom_commune` aussi, et dans mon fix j'ai rajouté "Non renseigné" (ou non dispo) car ça produit des adresses bizarre et incomplètes sinon